docs/source/languagesections/languagesections.js

```diff
@@ -52,24 +52,27 @@ $(function() {
   $('div.code-section ul.section-selector li,div.plain-section ul.section-selector li').click(function(evt) {
     evt.preventDefault();
 
-    var sel_class = $(this).attr('class').replace(' docutils container', '').replace('notranslate', '');
+    var sel_class = $(this).attr('class')
+      .replace(' docutils container', '')
+      .replace('notranslate', '')
+      .replace(' selected', '');
 
     $('ul.section-selector li').each(function() {
       var parent = $(this).parent().parent();
-      
+      var my_sel_class = sel_class;
       // When the target language is not available, default to bash or python.
       if (!$('div.' + sel_class, parent).length) {
-        if ($('div.' + 'highlight-bash', parent).length)
-          sel_class = 'highlight-bash';
+        if ($('div.bash', parent).length)
+          my_sel_class = 'bash';
```

Comment:
This was mutating the global-er `sel_class` per iteration loop, so we'd switch permanently to bash/python if available.